openwebstart/src/main/java/com/openwebstart/jvm/ui/dialogs/DialogWithResult.java

```diff
@@ -0,0 +1,79 @@
+package com.openwebstart.jvm.ui.dialogs;
+
+import com.openwebstart.jvm.runtimes.RemoteJavaRuntime;
+import com.openwebstart.jvm.ui.Images;
+import com.openwebstart.jvm.ui.util.IconComponent;
+import net.adoptopenjdk.icedteaweb.Assert;
+
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.ImageIcon;
+import javax.swing.JButton;
+import javax.swing.JDialog;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import java.awt.BorderLayout;
+import java.util.Arrays;
+
+public class DialogWithResult<R> extends JDialog {
+
+    private R result;
+
+    public DialogWithResult(final String title, final String message,DialogButton<R>... buttons) {
```

Comment:
space and final before `DialogButton<R>`

Reply:
Done